src/NUnitFramework/framework/Api/FrameworkPackageSettings.cs

```diff
@@ -106,10 +106,15 @@ public static class FrameworkPackageSettings
         public const string DefaultTestNamePattern = "DefaultTestNamePattern";
 
         /// <summary>
-        /// Parameters to be passed on to the test
+        /// Parameters to be passed on to the tests, serialized to a single string which needs parsing. Obsoleted by <see cref="TestParametersDictionary"/>; kept for backward compatibility.
         /// </summary>
         public const string TestParameters = "TestParameters";
 
+        /// <summary>
+        /// Parameters to be passed on to the tests, already parsed into an IDictionary&lt;string, string>. Replaces <see cref="TestParameters"/>.
```

Comment:
Just a little typo here. Not important enough to hold up the release, but perhaps can be fixed if you get back to it before it's merged, @jnm2 . 😄